### PR TITLE
[nl] Machine translation of missing keys

### PR DESCRIPTION
**Pull request by andrey-helldar**

### CODE REVIEW

**Comment by caouecs** [comment]
@WhereIsLucas can you help us and review this automatic pull request, please ? thanks by advance

File: locales/nl/nl.json [hunk 1]
```diff
@@ -476,23 +476,23 @@
...
     "South Africa": "Zuid Afrika",
-    "South Georgia And Sandwich Isl.": "South Georgia and South Sandwich Islands",
+    "South Georgia And Sandwich Isl.": "Zuid-Georgië en Zuidelijke Sandwicheilanden",
     "South Sudan": "Zuid Soedan",
-    "Spain": "Spain",
+    "Spain": "Spanje",
     "Sri Lanka": "Sri Lanka",
-    "Start Polling": "Start Polling",
-    "Stop Polling": "Stop Polling",
+    "Start Polling": "Begin Met Stemmen",
```
Comment (WhereIsLucas) [inline locales/nl/nl.json:483]: There's no word for polling, I think we should use the english work

**Review by WhereIsLucas — CHANGES_REQUESTED** [review]

File: locales/nl/nl.json [hunk 2]
```diff
@@ -24,115 +24,115 @@
...
+    "Choose date": "Kies datum",
+    "Choose File": "Bestand Kiezen",
+    "Choose Type": "Kies Type",
+    "Christmas Island": "Kerst Eiland",
     "click here to request another": "vraag hier een andere aan",
-    "Click to choose": "Click to choose",
+    "Click to choose": "Klik om te kiezen",
     "Close": "Sluit",
-    "Cocos (Keeling) Islands": "Cocos (Keeling) Islands",
+    "Cocos (Keeling) Islands": "Capaciteit Van De Levering: 10000pcs \/ Month",
```
[the remark this answers is not in the document]
Comment (andrey-helldar) [reply locales/nl/nl.json:118]: Yandex ¯\_(ツ)_/¯

File: locales/nl/nl.json [hunk 3]
```diff
@@ -476,23 +476,23 @@
...
-    "South Georgia And Sandwich Isl.": "South Georgia and South Sandwich Islands",
+    "South Georgia And Sandwich Isl.": "Zuid-Georgië en Zuidelijke Sandwicheilanden",
     "South Sudan": "Zuid Soedan",
-    "Spain": "Spain",
+    "Spain": "Spanje",
     "Sri Lanka": "Sri Lanka",
-    "Start Polling": "Start Polling",
-    "Stop Polling": "Stop Polling",
+    "Start Polling": "Begin Met Stemmen",
+    "Stop Polling": "Stop De Peilingen",
```
Comment (andrey-helldar) [inline locales/nl/nl.json:484]: @WhereIsLucas, english too?

File: locales/nl/nl.json [hunk 4]
```diff
@@ -4,7 +4,7 @@
     "90 Days": "90 dagen",
     ":amount Total": ":amount Totaal",
     ":resource Details": ":resource Details",
-    ":resource Details: :title": ":resource Details: :title",
+    ":resource Details: :title": "Details: :title",
```
Comment (caouecs) [inline locales/nl/nl.json:7]: Why :resource was deleted ?

Comment (caouecs) [reply locales/nl/nl.json:7]: I think it's : ":resource Details: :title"

**Review by caouecs — CHANGES_REQUESTED** [review]
Little fix to do

File: locales/nl/nl.json (hunk 4 above)
Comment (andrey-helldar) [reply locales/nl/nl.json:7]: Wow

**Review by caouecs — APPROVED** [review]

File: locales/nl/nl.json [hunk 5]
```diff
@@ -3,8 +3,8 @@
     "60 Days": "60 dagen",
     "90 Days": "90 dagen",
     ":amount Total": ":amount Totaal",
-    ":resource Details": ":resource Details",
-    ":resource Details: :title": ":resource Details: :title",
+    ":resource Details": ":resource Informatie",
```
Comment (andrey-helldar) [inline locales/nl/nl.json:6]: Google does not translate the word "Details" into Dutch. Therefore, I took as a basis the translated word "Details" from [file](https://github.com/Laravel-Lang/lang/pull/1644/files#diff-956387d70e3715ae6821373868ebb510d93dfc1570169cad7e09e9f412aeb62dL163).

@WhereIsLucas, can you tell i did the right thing?

image

**Comment by caouecs** [comment]
I merge and after I fix some details

**Pull request merged**